lib/file_validator.rb

```diff
@@ -77,9 +77,15 @@ def expected_files(scope)
       expected_files = []
       expected_files << "files.tgz" if has_files_tarball
       expected_files += tree_tarballs
+    elsif scope == "changed_managed_files"
+      expected_files = files.
+        reject do |file|
+          file["changes"].include?("deleted") ||
+          (file["type"] && file["type"] != "file")
+        end.map { |file| file["name"] }
     else
       expected_files = files.
-        reject { |file| file["changes"].include?("deleted") }.
+        reject { |file | file["changes"].include?("deleted") }.
```

Comment:
Align the operands of an expression in an assignment spanning multiple lines.<br>Space after last block parameter detected.